README.md

```diff
@@ -12,6 +12,7 @@
 
 - [Applications](#applications)
 	- [Command-line](#command-line)
+	- [Aixt](https://github.com/fermarsan/aixt)
```

Comment:
This doesn't look like the correct place.  It should go down in the command line section, not in the TOC.